specs/Standards/Tokens/FungibleToken.md

```diff
@@ -173,13 +210,20 @@ pub fn set_allowance(&mut self, escrow_account_id: AccountId, allowance: U128);
 /// * If this function is called by an escrow account (`owner_id != predecessor_account_id`),
 ///   then the allowance of the caller of the function (`predecessor_account_id`) on
 ///   the account of `owner_id` should be greater or equal than the transfer `amount`.
+/// * Caller of the method has to attach deposit enough to cover storage difference at the
+///   fixed storage price defined in the contract.
+#[payable]
 pub fn transfer_from(&mut self, owner_id: AccountId, new_owner_id: AccountId, amount: U128);
```

Comment:
How does this function lead to an increase in state?

Reply:
If the account for the new owner is created